packages/astro/src/core/ssr/index.ts

```diff
@@ -133,7 +133,7 @@ export async function preload({ astroConfig, filePath, viteServer }: SSROptions)
   // Important: This needs to happen first, in case a renderer provides polyfills.
   const renderers = await resolveRenderers(viteServer, astroConfig);
   // Load the module from the Vite SSR Runtime.
-  const mod = (await viteServer.ssrLoadModule(viteifyURL(filePath))) as ComponentInstance;
+  const mod = (await viteServer.ssrLoadModule(fileURLToPath(filePath))) as ComponentInstance;
```

Comment:
Add a note about path being the correct input here?

Reply:
Let me double-check this on Windows actually. Not documented here, but the original change to `ssrLoadModule()` had to do with a hack for `transformIndexHtml()` that we don’t need anymore (moving to project-relative URLs in the compiler and in SSR fixed the issues).

But let me test this on Windows just to make sure (it’s the client-side hydration that this affects)

Reply:
OK this works on Windows! Not worth going into detail but the original change here wasn’t actually needed, again, the switch to relative URLs fixed what this somewhat-hack did. Moving to `fileURLToPath()` is just more straightforward.